doc/starting_bft_blockchain.md

```diff
@@ -24,8 +24,9 @@ Utilising a Round Robin algorithm.
 
 ```yaml
 blockchain_configuration:
-  - [ block0-date, 1550822014 ]
-  - [ discrimination, test ]
+  block0_date: 1550822014
```

Comment:
Less weird data format, yay!

Reply:
thanks to @CodeSandwich for this new model, it's based on his plan described in rust-cardano#610 . he will update it to better format later.